src/Umbraco.Core/Models/Mapping/MemberMapDefinition.cs

```diff
@@ -0,0 +1,26 @@
+using Umbraco.Core.Mapping;
+using Umbraco.Web.Models.ContentEditing;
+
+namespace Umbraco.Core.Models.Mapping
+{
+    /// <inheritdoc />
+    public class MemberMapDefinition : IMapDefinition
+    {
+        /// <inheritdoc />
+        public void DefineMaps(UmbracoMapper mapper) => mapper.Define<MemberSave, IMember>(Map);
+
+        //TODO: put this here instead of a new mapper definition (like user). Can move
```

Comment:
@Shazwazza @bergmania I needed to map the MemberSave (content item member) to a member, so I added this definition, although it doesn't yet have groups.

I didn't know if this was the right place to put it, so I've flagged it with a TODO to make sure I'm mapping everything correctly.

Reply:
Seems correct to me.  Both `MemberSave` and `IMember` exists in Umbraco.Core